packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,294 @@ components:
             created
           example: 1587667174725
 
+    asset:
+      type: object
+      table: asset
+      additionalProperties: false
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        type:
+          type: string
+          enum:
+            - video
+            - audio
+          description: Type of the asset.
+          example: video
+        playbackId:
+          type: string
+          example: eaw4nk06ts2d0mzb
+          description: Used to form playback URL and storage folder
+        userId:
+          type: string
+          readOnly: true
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          writeOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description:
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        meta:
+          type: object
+          description: User input metadata associated with the asset
+          example:
+            {
+              "title": "My awesome video",
+              "description": "This is a video of my awesome life",
+              "tags": ["awesome", "life", "video"],
+            }
+        updatedAt:
+          type: number
+          description:
+            Timestamp (in milliseconds) at which the asset was last updated
+          example: 1587667174725
+        createdAt:
+          type: number
+          readOnly: true
+          description: Timestamp (in milliseconds) at which asset was created
+          example: 1587667174725
+        hash:
+          type: array
+          description: Hash of the asset
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              hash:
```

Comment:
Reminder for myself that I need a third processing of the file contents for calculating this hash. Or get it from the response from SE, but I'm not sure if it's available (currently not exposed on the OS drivers lib at least)